package.json

```diff
@@ -40,6 +40,7 @@
     "react-native-multiple-select": "^0.5.5",
     "react-native-safe-area-context": "~3.0.7",
     "react-native-screens": "~2.9.0",
+    "react-native-vector-icons": "^7.1.0",
```

Comment:
@ampsteric You still haven't to remove this `react-native-vector-icons` package and delete package-lock.json as well(we use yarn, so we commit yarn.lock).